### ISSUE TITLE
How to handle one off verbs and views?

### ISSUE THREAD

**Issue by sirwolfgang** [apiaryio/api-blueprint#142]
I apologize if I missed this in the documentation. I am trying to find a good way to express one off paths for verbs and other views. In these cases, examples below, I don't think it makes sense to separate them by a header, but am required to in order to specify the URL. 

```
# Group Items
## Items [/items]
### Retrieve Items [GET]

## Item [/items/{id}]
### Retrieve an Item [GET]

## SKUs [/items/sku/{sku_number}]
### Retrieve an Item by SKU [GET]

## Inventories [/items/{id}/inventories]
### Retrieve an Item's Inventories [GET]

## Events [/items/{id}/events]
### Retrieve an Item's Events [GET]

## Warehouses [/items/{id}/warehouses]
### Retrieve an Item's Warehouses [GET]
```

**Comment by Anahkiasen** [comment]
Wondering the same thing, there should be a way to name one-off actions, like:

```md
# Get an user [GET /users/{user}]
```

Or something

**Comment by pksunkara** [comment]
@sirwolfgang @Anahkiasen Do you guys think you can make use of the features proposed in #88?

**Comment by Anahkiasen** [comment]
Would work for me yup

**Comment by zdne** [comment]
Addressed in #88.
